go.mod

```diff
@@ -1,74 +1,81 @@
 module github.com/superbrothers/ksort
 
-go 1.19
+go 1.22.0
+
+toolchain go1.22.6
```

Comment:
```suggestion
go 1.22.6
```

Reply:
`go mod tidy` added this automatically. Is there an option I should be using to avoid that?